x-pack/plugins/global_search_bar/public/search_syntax/parse_search_params.ts

```diff
@@ -17,32 +17,24 @@ const aliasMap = {
 };
 
 export const parseSearchParams = (term: string): ParsedSearchParams => {
+  const recognizedFields = knownFilters.concat(...Object.values(aliasMap));
   let query: Query;
 
   try {
-    query = Query.parse(term);
+    query = Query.parse(term, {
+      schema: { recognizedFields },
+    });
   } catch (e) {
     // if the query fails to parse, we just perform the search against the raw search term.
     return {
       term,
-      filters: {
-        unknowns: {},
-      },
+      filters: {},
     };
   }
 
   const searchTerm = getSearchTerm(query);
   const filterValues = applyAliases(getFieldValueMap(query), aliasMap);
 
-  const unknownFilters = [...filterValues.entries()]
```

Comment:
In the previous code, nothing was handling the `unknowns` field from the result. This PR takes the opinion that pulling out unknown field clauses and making them unusable is not the user's intention.